[BUG] Item's bug out When Hud is turned off

### LiquidBounce Branch

Nextgen

### LiquidBounce Build/Version

git-10c0984

### Operating System

Windows 11

### Minecraft Version

1.20.4

### Describe the bug

![image](https://github.com/CCBlueX/LiquidBounce/assets/101041312/b62dea11-aad1-49b7-82e3-3832cc060225)


### Steps to reproduce

Turn client hud off.

### Client Log

```shell
[19:57:56] [main/INFO]: Loading Minecraft 1.20.4 with Fabric Loader 0.15.3
[19:57:56] [ForkJoinPool-1-worker-5/WARN]: Mod org_cloudburstmc_netty_netty-transport-raknet uses the version 1.0.0.CR1-SNAPSHOT which isn't compatible with Loader's extended semantic version format (Could not parse version number component 'CR1'!), SemVer is recommended for reliably evaluating dependencies and prioritizing newer version
[19:57:57] [main/INFO]: Loading 97 mods:
	- baritone 1.10.2
	   \-- dev_babbaj_nether-pathfinder 1.4.1
	- fabric-api 0.91.3+1.20.4
	   |-- fabric-api-lookup-api-v1 1.6.45+78d798af4f
	   |-- fabric-biome-api-v1 13.0.15+78d798af4f
	   |-- fabric-block-api-v1 1.0.14+78d798af4f
	   |-- fabric-block-view-api-v2 1.0.3+78d798af4f
	   |-- fabric-blockrenderlayer-v1 1.1.45+78d798af4f
	   |-- fabric-client-tags-api-v1 1.1.6+78d798af4f
	   |-- fabric-command-api-v1 1.2.40+f71b366f4f
	   |-- fabric-commands-v0 0.2.57+df3654b34f
	   |-- fabric-containers-v0 0.1.81+df3654b34f
	   |-- fabric-content-registries-v0 5.0.9+78d798af4f
	   |-- fabric-convention-tags-v1 1.5.9+78d798af4f
	   |-- fabric-crash-report-info-v1 0.2.22+78d798af4f
	   |-- fabric-data-generation-api-v1 13.1.17+78d798af4f
	   |-- fabric-dimensions-v1 2.1.60+78d798af4f
	   |-- fabric-entity-events-v1 1.5.28+4ced05924f
	   |-- fabric-events-interaction-v0 0.6.14+78d798af4f
	   |-- fabric-events-lifecycle-v0 0.2.71+df3654b34f
	   |-- fabric-game-rule-api-v1 1.0.45+78d798af4f
	   |-- fabric-item-api-v1 2.1.34+78d798af4f
	   |-- fabric-item-group-api-v1 4.0.20+78d798af4f
	   |-- fabric-key-binding-api-v1 1.0.40+78d798af4f
	   |-- fabric-keybindings-v0 0.2.38+df3654b34f
	   |-- fabric-loot-api-v2 2.1.4+78d798af4f
	   |-- fabric-message-api-v1 6.0.4+78d798af4f
	   |-- fabric-mining-level-api-v1 2.1.59+78d798af4f
	   |-- fabric-model-loading-api-v1 1.0.7+78d798af4f
	   |-- fabric-models-v0 0.4.6+9386d8a74f
	   |-- fabric-object-builder-api-v1 13.0.7+78d798af4f
	   |-- fabric-recipe-api-v1 2.0.15+78d798af4f
	   |-- fabric-renderer-api-v1 3.2.3+78d798af4f
	   |-- fabric-renderer-indigo 1.5.3+78d798af4f
	   |-- fabric-renderer-registries-v1 3.2.50+df3654b34f
	   |-- fabric-rendering-data-attachment-v1 0.3.41+73761d2e4f
	   |-- fabric-rendering-fluids-v1 3.0.32+78d798af4f
	   |-- fabric-rendering-v0 1.1.53+df3654b34f
	   |-- fabric-rendering-v1 3.0.12+78d798af4f
	   |-- fabric-resource-conditions-api-v1 2.3.13+78d798af4f
	   |-- fabric-screen-api-v1 2.0.16+78d798af4f
	   |-- fabric-screen-handler-api-v1 1.3.50+78d798af4f
	   |-- fabric-sound-api-v1 1.0.16+78d798af4f
	   |-- fabric-transfer-api-v1 4.0.6+78d798af4f
	   \-- fabric-transitive-access-wideners-v1 5.0.13+78d798af4f
	- fabric-language-kotlin 1.10.17+kotlin.1.9.22
	   |-- org_jetbrains_kotlin_kotlin-reflect 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk7 1.9.22
	   |-- org_jetbrains_kotlin_kotlin-stdlib-jdk8 1.9.22
	   |-- org_jetbrains_kotlinx_atomicfu-jvm 0.23.1
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm 1.7.3
	   |-- org_jetbrains_kotlinx_kotlinx-coroutines-jdk8 1.7.3
	   |-- org_jetbrains_kotlinx_kotlinx-datetime-jvm 0.5.0
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-cbor-jvm 1.6.2
	   |-- org_jetbrains_kotlinx_kotlinx-serialization-core-jvm 1.6.2
	   \-- org_jetbrains_kotlinx_kotlinx-serialization-json-jvm 1.6.2
	- fabricloader 0.15.3
	   \-- mixinextras 0.3.2
	- java 17
	- liquidbounce 0.1.5
	- minecraft 1.20.4
	- moddetectionpreventer 1.0.0
	- modmenu 9.0.0-pre.1
	- nochatreports 1.20.4-v2.5.0
	- sodium 0.5.5
	- viafabricplus 3.0.6
	   |-- com_google_code_findbugs_jsr305 3.0.2
	   |-- com_vdurmont_semver4j 3.1.0
	   |-- com_viaversion_viabackwards-common 4.10.0-23w51b-SNAPSHOT
	   |-- com_viaversion_viaversion-common 4.10.0-23w51b-SNAPSHOT
	   |-- de_florianmichael_classic4j 2.0.1
	   |-- fabric-api-base 0.4.36+78d798af4f
	   |-- fabric-command-api-v2 2.2.20+78d798af4f
	   |-- fabric-lifecycle-events-v1 2.2.30+78d798af4f
	   |-- fabric-networking-api-v1 3.1.5+b7e146354f
	   |-- fabric-particles-v1 1.1.7+78d798af4f
	   |-- fabric-registry-sync-v0 4.0.16+78d798af4f
	   |-- fabric-resource-loader-v0 0.11.16+78d798af4f
	   |-- io_jsonwebtoken_jjwt-api 0.12.3
	   |-- io_jsonwebtoken_jjwt-gson 0.12.3
	   |-- io_jsonwebtoken_jjwt-impl 0.12.3
	   |-- net_jodah_expiringmap 0.5.10
	   |-- net_lenni0451_mcping 1.4.0
	   |-- net_lenni0451_mcstructs-bedrock_forms 1.2.0
	   |-- net_lenni0451_mcstructs-bedrock_text 1.2.0
	   |-- net_lenni0451_reflect 1.3.1
	   |-- net_raphimc_minecraftauth 3.1.0
	   |-- net_raphimc_viaaprilfools 2.0.11-SNAPSHOT
	   |-- net_raphimc_viabedrock 0.0.4-SNAPSHOT
	   |-- net_raphimc_vialegacy 2.2.22-SNAPSHOT
	   |-- net_raphimc_vialoader 2.2.13-SNAPSHOT
	   |-- org_cloudburstmc_netty_netty-transport-raknet 1.0.0.CR1-SNAPSHOT
	   |-- org_iq80_leveldb_leveldb 0.12
	   |-- org_iq80_leveldb_leveldb-api 0.12
	   |-- org_lz4_lz4-pure-java 1.8.0
	   \-- org_yaml_snakeyaml 2.2
[19:57:57] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.5 Source=file:/C:/Users/razzy/AppData/Roaming/CCBlueX/LiquidLauncher/data/libraries/net/fabricmc/sponge-mixin/0.12.5+mixin.0.8.5/sponge-mixin-0.12.5+mixin.0.8.5.jar Service=Knot/Fabric Env=CLIENT
[19:57:57] [main/INFO]: Compatibility level set to JAVA_17
[19:57:58] [main/INFO]: Loaded configuration file for Sodium: 42 options available, 0 override(s) found
[19:57:59] [main/INFO]: Searching for graphics cards...
[19:58:00] [main/INFO]: Found graphics card: GraphicsAdapterInfo[vendor=NVIDIA, name=NVIDIA GeForce GTX 1660, version=DriverVersion=31.0.15.3640]
[19:58:00] [main/WARN]: Sodium has applied one or more workarounds to prevent crashes or other issues on your system: [NVIDIA_THREADED_OPTIMIZATIONS]
[19:58:00] [main/WARN]: This is not necessarily an issue, but it may result in certain features or optimizations being disabled. You can sometimes fix these issues by upgrading your graphics driver.
[19:58:00] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.3.2).
[19:58:03] [main/INFO]: liquidbounce.mixins.json:minecraft.entity.MixinPlayerEntityAccessor from mod liquidbounce->@Accessor[FIELD_GETTER]::getTrackedMainArm()Lnet/minecraft/class_2940; should be static as its target is
[19:58:09] [Datafixer Bootstrap/INFO]: 198 Datafixer optimizations took 176 milliseconds
[19:58:12] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[19:58:13] [Render thread/INFO]: Setting user: Rafs
[19:58:13] [Via-Mappingloader-0/INFO]: Loading block connection mappings ...
[19:58:13] [Download-2/ERROR]: Failed to fetch user properties
com.mojang.authlib.exceptions.InvalidCredentialsException: Status: 401
	at com.mojang.authlib.exceptions.MinecraftClientHttpException.toAuthenticationException(MinecraftClientHttpException.java:59) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.fetchProperties(YggdrasilUserApiService.java:150) ~[authlib-6.0.52.jar:?]
	at net.minecraft.class_310.method_55608(class_310.java:467) ~[client-intermediary.jar:?]
	at java.util.concurrent.CompletableFuture$AsyncSupply.run(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source) ~[?:?]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source) ~[?:?]
	at java.lang.Thread.run(Unknown Source) ~[?:?]
Caused by: com.mojang.authlib.exceptions.MinecraftClientHttpException: Status: 401
	at com.mojang.authlib.minecraft.client.MinecraftClient.readInputStream(MinecraftClient.java:100) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.minecraft.client.MinecraftClient.get(MinecraftClient.java:57) ~[authlib-6.0.52.jar:?]
	at com.mojang.authlib.yggdrasil.YggdrasilUserApiService.fetchProperties(YggdrasilUserApiService.java:121) ~[authlib-6.0.52.jar:?]
	... 5 more
[19:58:13] [Via-Mappingloader-0/INFO]: Using FastUtil Long2ObjectOpenHashMap for block connections
[19:58:14] [ForkJoinPool.commonPool-worker-1/INFO]: Loading translations...
[19:58:14] [ForkJoinPool.commonPool-worker-1/INFO]: Registering protocols...
[19:58:14] [Render thread/INFO]: [Indigo] Different rendering plugin detected; not applying Indigo.
[19:58:14] [Render thread/WARN]: ClothConfig API not found, cannot provide config screen factory.
[19:58:14] [Worker-Main-1/INFO]: Checking mod updates...
[19:58:15] [Worker-Main-1/INFO]: Update available for 'fabric-api@0.91.3+1.20.4', (-> 0.96.4+1.20.4)
[19:58:15] [Worker-Main-1/INFO]: Update available for 'fabric-language-kotlin@1.10.17+kotlin.1.9.22', (-> 1.10.19+kotlin.1.9.23)
[19:58:15] [Worker-Main-1/INFO]: Update available for 'moddetectionpreventer@1.0.0', (-> 1.1.0)
[19:58:15] [Worker-Main-1/INFO]: Update available for 'sodium@0.5.5', (-> mc1.20.4-0.5.8)
[19:58:15] [Worker-Main-1/INFO]: Update available for 'modmenu@9.0.0-pre.1', (-> 9.0.0)
[19:58:15] [Render thread/INFO]: Backend library: LWJGL version 3.3.2-snapshot
[19:58:15] [Render thread/WARN]: Applying workaround: Prevent the NVIDIA OpenGL driver from using broken optimizations (NVIDIA_THREADED_OPTIMIZATIONS)
[19:58:16] [Render thread/INFO]: OpenGL Vendor: NVIDIA Corporation
[19:58:16] [Render thread/INFO]: OpenGL Renderer: NVIDIA GeForce GTX 1660/PCIe/SSE2
[19:58:16] [Render thread/INFO]: OpenGL Version: 3.3.0 NVIDIA 536.40
[19:58:16] [ForkJoinPool.commonPool-worker-1/INFO]: Started resource pack HTTP server on http://127.0.0.1:30594/
[19:58:17] [ForkJoinPool.commonPool-worker-1/INFO]: ViaVersion detected lowest supported version by the proxy: 1.7.2-1.7.5 (4)
[19:58:17] [ForkJoinPool.commonPool-worker-1/INFO]: Highest supported version by the proxy: 23w51b (766)
[19:58:17] [ForkJoinPool.commonPool-worker-1/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[19:58:17] [Render thread/INFO]: Launching LiquidBounce v0.1.5 by CCBlueX
[19:58:18] [Render thread/INFO]: Loaded language en_us
[19:58:18] [Render thread/INFO]: Loaded language de_de
[19:58:18] [Render thread/INFO]: Loaded language ja_jp
[19:58:18] [Render thread/INFO]: Loaded language zh_cn
[19:58:18] [Render thread/INFO]: Loaded language ru_ru
[19:58:18] [Render thread/INFO]: Loaded language en_pt
[19:58:18] [Render thread/INFO]: Loaded language pt_br
[19:58:21] [Render thread/INFO]: Successfully extracted default theme
[19:58:21] [Render thread/INFO]: Inserted 9 components
[19:58:21] [Render thread/INFO]: Successfully loaded config 'enemies'.
[19:58:21] [Render thread/INFO]: (Chat) 
[19:58:21] [Render thread/INFO]: (Chat)  f lLiquid 9 lBounce  8?  7Auto ConfigAuto Config
[19:58:21] [Render thread/INFO]: Successfully loaded config 'modules'.
[19:58:21] [Render thread/INFO]: Successfully loaded config 'commands'.
[19:58:21] [Render thread/INFO]: Successfully loaded config 'friends'.
[19:58:21] [Render thread/ERROR]: Unable to deserialize value proxy
com.google.gson.JsonSyntaxException: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:397) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1227) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1329) ~[gson-2.10.1.jar:?]
	at com.google.gson.Gson.fromJson(Gson.java:1271) ~[gson-2.10.1.jar:?]
	at net.ccbluex.liquidbounce.config.Value.deserializeFrom(Value.kt:218) ~[LiquidBounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeValue(ConfigSystem.kt:273) ~[LiquidBounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:228) ~[LiquidBounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable(ConfigSystem.kt:203) ~[LiquidBounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.deserializeConfigurable$default(ConfigSystem.kt:201) ~[LiquidBounce.jar:?]
	at net.ccbluex.liquidbounce.config.ConfigSystem.load(ConfigSystem.kt:144) ~[LiquidBounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:161) ~[LiquidBounce.jar:?]
	at net.ccbluex.liquidbounce.LiquidBounce$startHandler$1.invoke(LiquidBounce.kt:111) ~[LiquidBounce.jar:?]
	at net.ccbluex.liquidbounce.event.EventManager.callEvent(EventManager.kt:181) ~[LiquidBounce.jar:?]
	at net.minecraft.class_310.handler$bmj000$liquidbounce$startClient(class_310.java:9351) ~[client-intermediary.jar:?]
	at net.minecraft.class_310.<init>(class_310.java:675) ~[client-intermediary.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:223) ~[fabric-loader-0.15.3-1.20.4.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:470) ~[fabric-loader-0.15.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74) ~[fabric-loader-0.15.3.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.15.3.jar:?]
Caused by: java.lang.IllegalStateException: Expected BEGIN_OBJECT but was BEGIN_ARRAY at path $
	at com.google.gson.internal.bind.JsonTreeReader.expect(JsonTreeReader.java:165) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.JsonTreeReader.beginObject(JsonTreeReader.java:89) ~[gson-2.10.1.jar:?]
	at com.google.gson.internal.bind.ReflectiveTypeAdapterFactory$Adapter.read(ReflectiveTypeAdapterFactory.java:386) ~[gson-2.10.1.jar:?]
	... 18 more
[19:58:21] [Render thread/INFO]: Successfully loaded config 'proxy'.
[19:58:21] [Render thread/INFO]: Environment: Environment[sessionHost=http://sessionserver.thealtening.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[19:58:21] [Render thread/INFO]: Environment: Environment[sessionHost=http://sessionserver.thealtening.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[19:58:21] [Render thread/INFO]: Environment: Environment[sessionHost=http://sessionserver.thealtening.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[19:58:21] [Render thread/INFO]: Successfully loaded config 'accounts'.
[19:58:21] [Render thread/INFO]: Successfully loaded config 'tabs'.
[19:58:21] [Render thread/INFO]: Successfully loaded config 'lang'.
[19:58:21] [Render thread/INFO]: Successfully loaded config 'storage'.
[19:58:21] [netty-websocket/INFO]: Starting Netty server...
[19:58:21] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969] REGISTERED
[19:58:21] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969] BIND: 0.0.0.0/0.0.0.0:15259
[19:58:21] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] ACTIVE
[19:58:21] [netty-websocket/INFO]: Netty server started on port 15259.
[19:58:22] [Via Async Scheduler 0/INFO]: Finished mapping loading, shutting down loader executor!
[19:58:23] [Render thread/INFO]: java-cef commit: e91a133673f9524c5f7db9bdbad1aff61b3887d7
[19:58:23] [Render thread/INFO]: Task: Downloading Checksum with progress 0.0 %
[19:58:24] [Render thread/INFO]: Checksum matches: true
[19:58:24] [Render thread/INFO]: Platform directory exists: true
[19:58:24] [Render thread/INFO]: Initializing CEF on windows_amd64...
[19:58:24] [Render thread/INFO]: [STDOUT]: initialize on Thread[Render thread,5,main] with library path C:\Users\razzy\AppData\Roaming\CCBlueX\LiquidLauncher\data\gameDir\nextgen\LiquidBounce\mcef\libraries\windows_amd64
[19:58:24] [Render thread/INFO]: Chromium Embedded Framework initialized
[19:58:24] [Render thread/INFO]: Browser is ready.
[19:58:24] [Render thread/INFO]: Successfully loaded client!
[19:58:25] [Render thread/INFO]: Reloading ResourceManager: vanilla, fabric (com_viaversion_viabackwards-common, com_viaversion_viaversion-common, fabric-api, fabric-api-base, fabric-api-lookup-api-v1, fabric-biome-api-v1, fabric-block-api-v1, fabric-block-view-api-v2, fabric-blockrenderlayer-v1, fabric-client-tags-api-v1, fabric-command-api-v1, fabric-command-api-v2, fabric-commands-v0, fabric-containers-v0, fabric-content-registries-v0, fabric-convention-tags-v1, fabric-crash-report-info-v1, fabric-data-generation-api-v1, fabric-dimensions-v1, fabric-entity-events-v1, fabric-events-interaction-v0, fabric-events-lifecycle-v0, fabric-game-rule-api-v1, fabric-item-api-v1, fabric-item-group-api-v1, fabric-key-binding-api-v1, fabric-keybindings-v0, fabric-language-kotlin, fabric-lifecycle-events-v1, fabric-loot-api-v2, fabric-message-api-v1, fabric-mining-level-api-v1, fabric-model-loading-api-v1, fabric-models-v0, fabric-networking-api-v1, fabric-object-builder-api-v1, fabric-particles-v1, fabric-recipe-api-v1, fabric-registry-sync-v0, fabric-renderer-api-v1, fabric-renderer-indigo, fabric-renderer-registries-v1, fabric-rendering-data-attachment-v1, fabric-rendering-fluids-v1, fabric-rendering-v0, fabric-rendering-v1, fabric-resource-conditions-api-v1, fabric-resource-loader-v0, fabric-screen-api-v1, fabric-screen-handler-api-v1, fabric-sound-api-v1, fabric-transfer-api-v1, fabric-transitive-access-wideners-v1, fabricloader, liquidbounce, moddetectionpreventer, modmenu, net_raphimc_viaaprilfools, net_raphimc_viabedrock, net_raphimc_vialegacy, nochatreports, sodium, viafabricplus)
[19:58:25] [Render thread/INFO]: [CHAT] [Baritone] Baritone settings file not found, resetting.
[19:58:25] [Worker-Main-3/INFO]: Found unifont_all_no_pua-15.1.04.hex, loading
[19:58:25] [Render thread/INFO]: [STDOUT]: [nether-pathfinder] Created temp file at C:\Users\razzy\AppData\Local\Temp\nether_pathfinder-x86_6411020043060815870157.dll
[19:58:25] [Render thread/INFO]: [STDERR]: [nether-pathfinder] Failed to delete temp file
[19:58:25] [Render thread/INFO]: [STDOUT]: [nether-pathfinder] Loaded shared library
[19:58:25] [Render thread/INFO]: Created item group Heads
[19:58:25] [Render thread/INFO]: Created item group Exploits
[19:58:25] [Render thread/INFO]: Created item group Containers
[19:58:25] [Render thread/INFO]: Successfully setup tabs
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ: [id: 0xc3bb551d, L:/127.0.0.1:15259 - R:/127.0.0.1:30604]
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ COMPLETE
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ: [id: 0x5ee5201d, L:/127.0.0.1:15259 - R:/127.0.0.1:30605]
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ COMPLETE
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ: [id: 0xbce9f158, L:/127.0.0.1:15259 - R:/127.0.0.1:30606]
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ: [id: 0xca7f557f, L:/127.0.0.1:15259 - R:/127.0.0.1:30607]
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ COMPLETE
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ: [id: 0xbf7531f4, L:/127.0.0.1:15259 - R:/127.0.0.1:30608]
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ COMPLETE
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ: [id: 0xaea45851, L:/127.0.0.1:15259 - R:/127.0.0.1:30609]
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ COMPLETE
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ: [id: 0x0bb76e3c, L:/127.0.0.1:15259 - R:/127.0.0.1:30611]
[19:58:26] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ COMPLETE
[19:58:27] [Render thread/WARN]: Missing sound for event: minecraft:item.goat_horn.play
[19:58:27] [Render thread/WARN]: Missing sound for event: minecraft:entity.goat.screaming.horn_break
[19:58:27] [Render thread/WARN]: Missing sound for event: minecraft:entity.generic.wind_burst
[19:58:27] [Render thread/INFO]: OpenAL initialized on device OpenAL Soft on Headphones (2- HD 450BT)
[19:58:27] [Render thread/INFO]: Sound engine started
[19:58:27] [Render thread/INFO]: Created: 1024x512x4 minecraft:textures/atlas/blocks.png-atlas
[19:58:27] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/signs.png-atlas
[19:58:27] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/banner_patterns.png-atlas
[19:58:27] [Render thread/INFO]: Created: 512x512x4 minecraft:textures/atlas/shield_patterns.png-atlas
[19:58:27] [Render thread/INFO]: Created: 1024x1024x4 minecraft:textures/atlas/armor_trims.png-atlas
[19:58:27] [Render thread/INFO]: Created: 256x256x4 minecraft:textures/atlas/chest.png-atlas
[19:58:27] [Render thread/INFO]: Created: 128x64x4 minecraft:textures/atlas/decorated_pot.png-atlas
[19:58:27] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/beds.png-atlas
[19:58:27] [Render thread/INFO]: Created: 512x256x4 minecraft:textures/atlas/shulker_boxes.png-atlas
[19:58:28] [Render thread/WARN]: Shader rendertype_entity_translucent_emissive could not find sampler named Sampler2 in the specified shader program.
[19:58:28] [Render thread/INFO]: Created: 512x256x0 minecraft:textures/atlas/particles.png-atlas
[19:58:28] [Render thread/INFO]: Created: 256x256x0 minecraft:textures/atlas/paintings.png-atlas
[19:58:28] [Render thread/INFO]: Created: 128x128x0 minecraft:textures/atlas/mob_effects.png-atlas
[19:58:28] [Render thread/INFO]: Created: 1024x512x0 minecraft:textures/atlas/gui.png-atlas
[19:58:28] [Render thread/INFO]: Loading fonts...
[19:58:28] [Render thread/INFO]: Loading queued font Montserrat
[19:58:28] [Render thread/INFO]: Loaded fonts successfully!
[19:58:29] [Render thread/INFO]: Loading Discord IPC configuration...
[19:58:29] [Render thread/INFO]: Loaded Discord IPC configuration.
[19:58:29] [Render thread/INFO]: Refreshing local IP info...
[19:58:30] [nioEventLoopGroup-5-1/INFO]: IP Info [AU, AS18371 Neighbourhood Cable]
[19:58:30] [Render thread/INFO]: Loading heads...
[19:58:32] [Render thread/INFO]: Successfully loaded 956 heads from the database
[19:58:32] [Render thread/INFO]: Loading settings list from API...
[19:58:32] [Render thread/INFO]: Loaded 26 settings from API.
[19:58:32] [Render thread/INFO]: Successfully loaded 233 cape carriers.
[20:00:36] [Render thread/ERROR]: ########## GL ERROR ##########
[20:00:36] [Render thread/ERROR]: @ Render
[20:00:36] [Render thread/ERROR]: 65539: Invalid standard cursor 0x00000000
[20:00:38] [Render thread/ERROR]: ########## GL ERROR ##########
[20:00:38] [Render thread/ERROR]: @ Render
[20:00:38] [Render thread/ERROR]: 65539: Invalid standard cursor 0x00000000
[20:00:42] [DefaultDispatcher-worker-1/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[20:00:48] [Render thread/ERROR]: Can't ping play.minecave.org: Internal Exception: java.net.SocketException: Connection reset
[20:00:49] [Render thread/ERROR]: Can't ping pr2.pumpkraft.net: Internal Exception: io.netty.handler.codec.DecoderException: java.io.IOException: Packet status/0 (class_2924) was larger than I expected, found 93 bytes extra whilst reading packet 0
[20:00:52] [Server Pinger #4/ERROR]: Failed to ping server Minecraft Server
io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: no further information: play.multiculmp.fun/66.206.27.114:50031
Caused by: java.net.ConnectException: Connection refused: no further information
	at sun.nio.ch.Net.pollConnect(Native Method) ~[?:?]
	at sun.nio.ch.Net.pollConnectNow(Unknown Source) ~[?:?]
	at sun.nio.ch.SocketChannelImpl.finishConnect(Unknown Source) ~[?:?]
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:337) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:776) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562) ~[netty-transport-4.1.97.Final.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997) ~[netty-common-4.1.97.Final.jar:?]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) ~[netty-common-4.1.97.Final.jar:?]
	at java.lang.Thread.run(Unknown Source) ~[?:?]
[20:00:56] [Render thread/INFO]: Successfully saved config 'modules'.
[20:00:56] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@414ff447, url=http://127.0.0.1:15259/default/#/)
[20:00:56] [Render thread/INFO]: Connecting to eu.loyisa.cn, 25565
[20:00:59] [Render thread/INFO]: Successfully saved config 'modules'.
[20:00:59] [Render thread/INFO]: Reloading integration browser JcefTab to Route(theme=net.ccbluex.liquidbounce.web.theme.Theme@414ff447, url=http://127.0.0.1:15259/default/#/)
[20:00:59] [Render thread/INFO]: Started 6 worker threads
[20:00:59] [Render thread/INFO]: [STDOUT]: World seems to be a replay. Not loading Baritone cache.
[20:01:00] [Render thread/WARN]: Ignoring player info update for unknown player 869dc6d1-9f4f-4818-9d1e-7c92a209d859 ([UPDATE_DISPLAY_NAME])
[20:01:00] [Render thread/INFO]: [CHAT]        
[20:01:00] [Render thread/INFO]: [CHAT] [+] Razzy6
[20:01:00] [Render thread/INFO]: [STDOUT]: Retrying to load Baritone cache
[20:01:00] [Render thread/INFO]: [STDOUT]: Baritone world data dir: C:\Users\razzy\AppData\Roaming\CCBlueX\LiquidLauncher\data\gameDir\nextgen\baritone\eu.loyisa.cn\minecraft\overworld_256
[20:01:00] [Render thread/INFO]: [STDOUT]: Cached world directory: C:\Users\razzy\AppData\Roaming\CCBlueX\LiquidLauncher\data\gameDir\nextgen\baritone\eu.loyisa.cn\minecraft\overworld_256\cache
[20:01:00] [Render thread/INFO]: [STDOUT]: Would save waypoints to C:\Users\razzy\AppData\Roaming\CCBlueX\LiquidLauncher\data\gameDir\nextgen\baritone\eu.loyisa.cn\minecraft\overworld_256\waypoints
[20:01:00] [pool-11-thread-1/INFO]: [STDOUT]: Loading region 0,0 from disk C:\Users\razzy\AppData\Roaming\CCBlueX\LiquidLauncher\data\gameDir\nextgen\baritone\eu.loyisa.cn\minecraft\overworld_256\cache
[20:01:00] [pool-11-thread-1/INFO]: [STDOUT]: Loaded region successfully in 365ms
[20:01:00] [Render thread/INFO]: [CHAT] Diona >> Your client brand is fabric
[20:01:01] [Render thread/WARN]: Shader ui_blur could not find uniform named BlurDir in the specified shader program.
[20:01:02] [Render thread/INFO]: [CHAT] Diona >> Our Discord: https://discord.gg/fdmkfts
[20:01:02] [pool-11-thread-1/INFO]: [STDOUT]: Loading region 0,-1 from disk C:\Users\razzy\AppData\Roaming\CCBlueX\LiquidLauncher\data\gameDir\nextgen\baritone\eu.loyisa.cn\minecraft\overworld_256\cache
[20:01:02] [pool-11-thread-1/INFO]: [STDOUT]: Loaded region successfully in 43ms
[20:01:03] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ: [id: 0x6d36c964, L:/127.0.0.1:15259 - R:/127.0.0.1:30673]
[20:01:03] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ COMPLETE
[20:01:03] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ: [id: 0x45b191e2, L:/127.0.0.1:15259 - R:/127.0.0.1:30674]
[20:01:03] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ COMPLETE
[20:01:12] [Render thread/INFO]: [CHAT] [-] Ovehimz
[20:01:13] [Worker-Main-4/INFO]: Successfully loaded cape for x0q_
[20:01:18] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ: [id: 0x649965c1, L:/127.0.0.1:15259 - R:/127.0.0.1:30682]
[20:01:18] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ COMPLETE
[20:01:30] [pool-11-thread-2/INFO]: [STDOUT]: Saving region 0,-1 to disk C:\Users\razzy\AppData\Roaming\CCBlueX\LiquidLauncher\data\gameDir\nextgen\baritone\eu.loyisa.cn\minecraft\overworld_256\cache
[20:01:30] [ForkJoinPool.commonPool-worker-2/INFO]: [STDOUT]: Saving region 0,0 to disk C:\Users\razzy\AppData\Roaming\CCBlueX\LiquidLauncher\data\gameDir\nextgen\baritone\eu.loyisa.cn\minecraft\overworld_256\cache
[20:01:30] [pool-11-thread-2/INFO]: [STDOUT]: Saved region successfully
[20:01:30] [ForkJoinPool.commonPool-worker-2/INFO]: [STDOUT]: Saved region successfully
[20:01:30] [pool-11-thread-2/INFO]: [STDOUT]: World save took 238ms
[20:01:37] [Render thread/INFO]: [CHAT]  f lLiquid 9 lBounce  8?  7HUD has been enabled.
[20:01:37] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ: [id: 0x39e757fe, L:/127.0.0.1:15259 - R:/127.0.0.1:30683]
[20:01:37] [nioEventLoopGroup-3-1/INFO]: [id: 0x63424969, L:/[0:0:0:0:0:0:0:0]:15259] READ COMPLETE
[20:01:39] [Render thread/INFO]: [CHAT] Razzy6: 24$@ @''<^ײ%# 4~   }   
[20:01:40] [Render thread/INFO]: [CHAT]  f lLiquid 9 lBounce  8?  7HUD has been disabled.
[20:01:45] [Render thread/INFO]: [CHAT] [-] Fyxar
[20:01:59] [Render thread/WARN]: Ignoring player info update for unknown player e324ca64-f338-4d36-bd36-0ceca3eaa39d ([UPDATE_GAME_MODE])
[20:01:59] [Render thread/WARN]: Ignoring player info update for unknown player e324ca64-f338-4d36-bd36-0ceca3eaa39d ([UPDATE_GAME_MODE])
[20:01:59] [Render thread/WARN]: Ignoring player info update for unknown player e324ca64-f338-4d36-bd36-0ceca3eaa39d ([UPDATE_DISPLAY_NAME])
[20:01:59] [Render thread/INFO]: [CHAT] [+] BlockHiter
[20:01:59] [Render thread/INFO]: [CHAT] [-] optimize_1
[20:02:02] [Render thread/INFO]: [CHAT]  f lLiquid 9 lBounce  8?  7Configs directory: C:\Users\razzy\AppData\Roaming\CCBlueX\LiquidLauncher\data\gameDir\nextgen\LiquidBounce\configs.
[20:02:16] [Render thread/WARN]: Ignoring player info update for unknown player 8894af5f-1a3d-495c-b1f0-2d62c975a1dc ([UPDATE_GAME_MODE])
[20:02:16] [Render thread/WARN]: Ignoring player info update for unknown player 8894af5f-1a3d-495c-b1f0-2d62c975a1dc ([UPDATE_DISPLAY_NAME])
[20:02:16] [Render thread/INFO]: [CHAT] [+] JoyTW
[20:02:45] [Render thread/WARN]: Ignoring player info update for unknown player 37e2eb70-db8a-41fd-bd40-219a359e951c ([UPDATE_GAME_MODE])
[20:02:45] [Render thread/WARN]: Ignoring player info update for unknown player 37e2eb70-db8a-41fd-bd40-219a359e951c ([UPDATE_GAME_MODE])
[20:02:45] [Render thread/WARN]: Ignoring player info update for unknown player 37e2eb70-db8a-41fd-bd40-219a359e951c ([UPDATE_DISPLAY_NAME])
[20:02:45] [Render thread/INFO]: [CHAT] [+] MiniWorldChe4ter
[20:03:00] [Render thread/INFO]: [CHAT] [-] JoyTW
[20:03:02] [Render thread/INFO]: [CHAT] [-] Alkaidhute
[20:03:09] [Render thread/INFO]: [CHAT] Diona >> Our Discord: https://discord.gg/fdmkfts
[20:03:21] [Render thread/INFO]: [CHAT] [-] BlockHiter
```


### Screenshots

_No response_

--- a/src/main/kotlin/net/ccbluex/liquidbounce/web/theme/component/ComponentOverlay.kt
+++ b/src/main/kotlin/net/ccbluex/liquidbounce/web/theme/component/ComponentOverlay.kt
@@ -42,8 +42,14 @@ object ComponentOverlay : Listenable {
         .any { it.enabled && it.tweaks.contains(tweak) }
 
     @JvmStatic
-    fun getComponentWithTweak(tweak: FeatureTweak) = components.filterIsInstance<IntegratedComponent>()
-        .find { it.enabled && it.tweaks.contains(tweak) }
+    fun getComponentWithTweak(tweak: FeatureTweak): IntegratedComponent? {
+        if (!handleEvents()) {
+            return null
+        }
+
+        return components.filterIsInstance<IntegratedComponent>()
+            .find { it.enabled && it.tweaks.contains(tweak) }
+    }
 
     fun insertComponents() {
         val componentList = ThemeManager.activeTheme.parseComponents()

